xpra/server/auth/keycloak.py

```diff
@@ -27,6 +30,9 @@ def __init__(self, **kwargs):
         self.client_id = kwargs.pop("client_id", KEYCLOAK_CLIENT_ID)
         self.client_secret_key = kwargs.pop("client_secret_key", KEYCLOAK_CLIENT_SECRET_KEY)
         self.redirect_uri = kwargs.pop("redirect_uri", KEYCLOAK_REDIRECT_URI)
+        self.claim_field = kwargs.pop("claim_field", KEYCLOAK_CLAIM_FIELD)
```

Comment:
Does this variable only drive the auth group check?
If so, wouldn't it be easier to remove it and check for `auth_condition` being set?

Reply:
No, this is used to store the list of keys used to retrieve the groups claim within userinfo, this is done with `extract_groups_claim()`.